docs/Compatibility.mdx

```diff
@@ -0,0 +1,13 @@
+---
+name: Compatibility
+---
+
+# Compatibility
+
+## Browser Support 
+
+Smooth-UI rely on [browsers supported by Styled Components](https://www.styled-components.com/docs/faqs#which-browsers-are-supported) or browsers supported by emotion (probably the same).
+
+As Smooth-UI is using v4.x of Styled Components, which is using React v16.3+, so it will support same browsers as the version of React and Styled Components.
```

Comment:
I think this paragraph brings nothing to the user.

Reply:
Can you remove it?

Reply:
Sure